librepo/downloader.c

```diff
@@ -1329,7 +1330,7 @@ check_finished_transfer_checksum(int fd,
                                               &calculated,
                                               err);
         if (!ret)
```

Comment:
To fix this, just replace:

    gboolean ret = lr_checksum_fd_compare(chksum->type,

with:

    ret = lr_checksum_fd_compare(chksum->type,